### ISSUE TITLE
Installing already installed R version

### ISSUE THREAD

**Issue by m-muecke** [r-lib/rig#235]
Not sure if this is the desired behaviour but when I run `rig add release` or  a specific version i.e. `rig add 4.4.0` and I've already installed the version with rig, then it still reinstalls the same version.

**Comment by gaborcsardi** [comment]
Yes, for now that is the desired behavior. It is sometimes hard to tell if the version to be installed is the same as the installed one or not. E.g. for patched versions. But sometimes even for released builds, e.g. the Linux builds have been patched to the RDS vulnerability.
